l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -256,7 +256,10 @@ def getCedPrest(self, cedPrest):
         if partner_id and not no_contact_update:
             partner_company_id = partner_model.browse(partner_id).company_id.id
             vals = {
-                'street': cedPrest.Sede.Indirizzo,
+                # mktsrl, 29/07/2021 - START: concatenated the XML house number node to the Address field
+                # 'street': cedPrest.Sede.Indirizzo,
+                'street': str(f"{cedPrest.Sede.Indirizzo} {str(cedPrest.Sede.NumeroCivico)}").strip(),
+                # mktsrl, 29/07/2021 - END: concatenated the XML house number node to the Address field
```

Comment:
Ciao, i commenti con le attribuzioni di autore e data delle modifiche e il vecchio codice non sono necessari, ci pensa git a tenerne traccia.

Reply:
Ok, chiaro @TheMule71. Mi potresti gentilmente dire che cosa devo fare?

Reply:
Rimuovi i commenti, lascia solo la riga 
```
'street': str(f"{cedPrest.Sede.Indirizzo} {str(cedPrest.Sede.NumeroCivico)}").strip(),
```

poi fai un:
```
git commit --amend
```
che ti fa modificare il commit precedente

poi fai:
```
git push -f
```

e questo "pusha" le modifiche sul tuo repo online, il -f è necessario per sovrascrivere il commit attuale. Non serve fare altro perché github si accorge della cosa e aggiorna la PR automaticamente. (in pratica la PR è collegata al branch nel tuo repo da quale è stata generata, ogni modifica a quel branch si riflette automaticamente sulla PR).

Reply:
Grazie @TheMule71 !! Confermo di aver seguito le indicazioni e aver aggiornato la PR